dhdt/auxilary/handler_randolph.py

```diff
@@ -1,99 +1,179 @@
 # functions to work with the Randolph Glacier Inventory
 import os
-import tempfile
-import numpy as np
 
-import geopandas
-
-from osgeo import ogr, osr, gdal
-
-from ..generic.handler_www import url_exist, get_zip_file, get_tar_file
-from ..generic.handler_sentinel2 import get_utmzone_from_tile_code, \
-    get_generic_s2_raster, get_geom_for_tile_code, get_bbox_from_tile_code, \
-    _check_mgrs_code
-
-def rgi_code2url(rgi_num, version=7):
+import affine
+import geopandas as gpd
+import pandas as pd
+import rasterio.features
+import rioxarray
```

Comment:
Even if not formally used, this sets up the `rio` accessor in xarray, used raster-based operations and for writing out the data